src/System.IO.Compression/src/System/IO/Compression/DeflateManaged/InflaterManaged.cs

```diff
@@ -45,8 +45,8 @@ internal sealed class InflaterManaged
 
         private readonly OutputWindow _output;
         private readonly InputBuffer _input;
-        private HuffmanTree _literalLengthTree;
-        private HuffmanTree _distanceTree;
+        private HuffmanTree _literalLengthTree = null!;
```

Comment:
Shouldn't this be nullable instead?

Reply:
These fields referenced without null check, seems by code logic it's always set before use, but yes its better to make it nullable and Assert whenever used, would update all similar fields below